src/viam/media/video.py

```diff
@@ -18,26 +27,43 @@ class RawImage(NamedTuple):
     mime_type: str
     """The mimetype of the image"""
 
-    width: int
-    """The width of the image"""
-
-    height: int
-    """The height of the image"""
-
     def close(self):
         """Close the image and release resources. For RawImage, this is a noop."""
         return
 
 
 class CameraMimeType(str, Enum):
-    RAW = "image/vnd.viam.rgba"
+    VIAM_RGBA = "image/vnd.viam.rgba"
     JPEG = "image/jpeg"
     PNG = "image/png"
     PCD = "pointcloud/pcd"
+    UNSUPPORTED = "unsupported"
+
+    @classmethod
+    def extract_from_lazy(cls, value: str) -> Tuple["CameraMimeType", bool]:
```

Comment:
Should we have a corresponding `@property` method to create a lazy mime type from an existing one? That way people can request a lazy image

Or should we have separate lazy enum members, e.g. `PNG_LAZY = "image/png+lazy"`?

Reply:
I think I would rather do the first one, but yeah some solution for this is necessary and I missed it